pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin.go

```diff
@@ -222,7 +223,8 @@ func applyToGateway(
 		}
 
 		if fromListenerRules, ok := rules.FromRules[listenerKey]; ok {
-			if err := configureInbound(fromListenerRules, proxy.Dataplane, listener, backends, path); err != nil {
+			conf := core_rules.ComputeConf[api.Conf](fromListenerRules, subsetutils.MeshElement())
```

Comment:
this reminded me, we should probably replace `from` with `rules` for MeshGateway as well? otherwise we won't be able to deprecate `spec.from`

Reply:
right, we will do this in separate pr: https://github.com/kumahq/kuma/issues/12729